cmd/daemon/player.go

```diff
@@ -151,21 +151,16 @@ func (p *AppPlayer) handlePlayerCommand(req dealer.RequestPayload) error {
 		p.state.setActive(true)
 		p.state.player.IsPlaying = false
 		p.state.player.IsBuffering = false
-		p.state.player.IsPaused = false
 
 		// options
 		p.state.player.Options = transferState.Options
 
 		// playback
+		// Note: this sets playback speed to 0 or 1 because that's all we're
+		// capable of, depending on whether the playback is paused or not.
 		p.state.player.Timestamp = transferState.Playback.Timestamp
 		p.state.player.PositionAsOfTimestamp = int64(transferState.Playback.PositionAsOfTimestamp)
-		p.state.player.IsPaused = transferState.Playback.IsPaused
-
-		if playbackSpeed := transferState.Playback.PlaybackSpeed; playbackSpeed != 0 {
-			p.state.player.PlaybackSpeed = playbackSpeed
-		} else {
-			p.state.player.PlaybackSpeed = 1
-		}
+		p.state.setPaused(transferState.Playback.IsPaused)
```

Comment:
I tried testing this, but switching playback over from Spotify Android (for example) to go-librespot appears to be somewhat broken even before this change.

Reply:
`no primary stream` sounds like a very strange issue for such small changes. Perhaps it is unrelated? `primaryStream` goes nil only when the device goes inactive or just before another track is loaded.

FYI, the logic removed here was required to fix a bug in the progress bar not moving (16741acf9823433324802d7e40624259a7ae99cf). I don't see a regression with this approch, just wanted to mention it.